integration-tests/src/test/kotlin/com/google/devtools/ksp/test/TestsCommon.kt

```diff
@@ -0,0 +1,66 @@
+import org.gradle.testkit.runner.GradleRunner
+import org.gradle.testkit.runner.TaskOutcome
+import org.junit.*
+import org.junit.Assert.assertEquals
+import org.junit.rules.TemporaryFolder
+import kotlin.jvm.JvmField
+import java.io.File
+import java.util.zip.ZipFile
+
+abstract class TestsCommon {
```

Comment:
I'd recommend implementing your own JUnit rule instead of using a base class. This makes composing different test fixtures easier in the long-term.

Reply:
Done. Thanks for the tips!